### ISSUE TITLE
Mod removes existing Immersive Engineering Treated Wood variants from Chisel GUI

### ISSUE THREAD

**Issue by SSyl** [asiekierka/UnlimitedChiselWorks#16]
When you normally open the Chisel GUI (without UnlimitedChiselWorks installed) and add Treated Wood Planks from Immersive Engineering, multiple variants show up in the GUI. However, with this mod installed those variants don't appear.

For example, Immersive Engineering has Treated Wood planks. Without Chisel, you can change those wood planks to their vertical variant or to their packaged variant by just placing them in a crafting grid. With Chisel installed by itself, when you open up the GUI and add Treated Wood Planks, it'll show you those three variants of the treated wood planks, as you can see here:

![2018-01-09_14 33 02](https://user-images.githubusercontent.com/6855315/34742332-69a0ef68-f54b-11e7-86fa-9cfc635747fc.png)

With UnlimitedChiselWorks, those three options get removed from the GUI and replaced with the ones your mod generates, as seen here:

![2018-01-09_14 37 38](https://user-images.githubusercontent.com/6855315/34742372-973775b4-f54b-11e7-9b62-f06745376316.png)

You can still get the Immersive Engineering variants by adding them to the crafting grid as you would without Chisel installed, but it would be nice if the variants were kept in the Chisel GUI as well.

I was worried this was affecting other mods as well, but I've tried Tinkers' Construct's seared bricks and Quarks Basalt and both were unaffected, so it seems as if it's only Immersive Engineering's treated wood.

TL;DR UnlimitedChiselWorks removes Immersive Engineering's existing Treated Wood Planks variants from the Chisel GUI.

**Comment by Hogan-Dazs** [comment]
It seems to not remove the existing IE chisel integration, it just ignores it and creates a new chiselGroup. Running CraftTweaker's `/ct chiselGroups` command reveals both the original `treated_wood` group and this mod's `immersiveengineering:treated_wood_0` group. This mod can fix the issue by utilizing the existing group, or it can be fixed with CT scripts that consolidate the two groups.
